metapackages/auto-instrumentations-node/test/register.test.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { promisify } from 'util';
+import * as childProcess from 'child_process';
+import * as assert from 'assert';
+
+const exec = promisify(childProcess.exec);
+
+describe('Register', function () {
+  this.timeout(5000);
+  it('can load auto instrumentation from command line', async () => {
+    const { stdout } = await exec(
+      'env OTEL_LOG_LEVEL=debug env OTEL_RESOURCE_DETECTORS=none node --require ./build/src/register.js ./test/test-app/app.js'
```

Comment:
Should we also add detectors here to verify they are working as expected?

Reply:
Same as the above comment. If it's not an issue to have a test that takes some time to finish, I prefer to test this as well.